README.md

```diff
@@ -1,45 +1,146 @@
+chmod +x litt-ubuntu-latest
+cp -f litt-ubuntu-latest /usr/local/bin/litt
+```
+- `wget` downloads the file (*make sure to change the version-number to the latest
+version*). 
+- `chmod +x` grants permission to execute `litt`. (This must be run as `sudo`
+  user)
+- `cp -f litt-ubuntu-latest /usr/local/bin/litt` makes `litt` available system wide under the name `litt`. 
+
+Verify by running `litt --version`. It should show something like: 
+```
+litt 1.0.1
+```
+
+#### MacOS
+Installation should work similar to Linux: 
+```
+wget https://github.com/georgbuechner/litt/releases/download/v1.0.1/litt-macOS-latest
+chmod +x litt-macOS-latest
+cp -f litt-macOS-latest /usr/local/bin/litt
+```
+
+#### Windows 
+Honestly, I don't really know. After downloading the file `litt` should be added
+to path. This guide gives some explanation on how to do that: 
+https://windowsloop.com/how-to-add-to-windows-path/
 
 ## Usage 
 
-### Create a new LITT-index 
+![basic functionality](images/basic_functionality.png)
+
+### Create a new `litt`-index <a name="create"></a>
 This is how you create a new index:
 ```
 litt <index-name> -i <path-to-documents>
 ```
-Assuming you have some Docuemts stored at `Documets/Literature/books/` which you
+Assuming you have some documents stored at `Documets/Literature/books/` which you
 would like to index, you can do this as following: 
 ```
 litt books -i Documets/Literature/books/
 ```
 *NOTE:*
 - *the index-name can be any name. It need not match with the directory name.*
-- *any relative path is automatically changed to an absolute path (e.i.
-  `Docuemts/Literature/books/` to `/home/<user>/Docuemts/Literature/books/`
+- *any relative path is automatically changed to an absolute path* (e.i.
+  `Docuemts/Literature/books/` to `/home/<user>/Docuemts/Literature/books/`)
+
+### Updating, reloading, deleting an existing index<a name="updating"></a>
+To see all existing indices, type: 
+```
+litt -l 
+> Currently available indices:
+ - ("books", "/home/<user>/Documents/Literature/books/")
+ - ("papers", "/home/<user>/Documents/papers/")
+ - ("notes", "/home/<user>/Documents/notes/")
+```
+You can then update an existing index: `litt books -u` which is usually very
+fast, but might not track all changes made to *existing* documents and will
+never track *deleted* documents. Use `litt books --reload` to fully reload the
+index. This might take a while. 
+
+To delete an index, type: `litt books -r`
 
 ### Searching 
 In general you search like this: 
 ```
 litt <index-name> <search-term>
 ```
-If your search term is more than one word, you should add quotations: `litt
-<index-name> '<term1 term2 ...>`
+If your search term is more than one word, you should add quotations: `litt <index-name> '<term1 term2 ...>'`
+
+Use `--offset` and `--limit` to show more results. (Default shows the top ten
+results. `--offset 10` shows the first 10 to 20 results. `--limit 50 --offset 10`
+shows the first 10 to 50 results).
```

Comment:
Wouldn't limit 50 and offset 50 mean that it's 10 to 60?

Reply:
Fixes in last commit